storage/stories/stories-de.json

```diff
@@ -1,4 +1,12 @@
 [
+  {
+    "id": "debug",
```

Comment:
debug story will be removed once all video features are done, i'm splitting it up in multiple PRs